requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/basxsoftwareassociation/htmlgenerator.git
+git+https://github.com/basxsoftwareassociation/bread.git
+git+https://github.com/basxsoftwareassociation/basxconnect.git
+# technically one entry "basxconnect" would be enough in order to use released packages, however, we currently want to use bleeding-edge versions
```

Comment:
For the requirements it might be reasonable to think whether we want to specify only the package names from pypi instead of the direct git-repo url. The demo would not need up to date with basxconnect master but only with the released packages.

Reply:
I just tried it with just basxconnect.
But then I get this error: 
`AttributeError: module 'htmlgenerator' has no attribute 'html_id'`
So I guess, we are not there yet. I will file an issue, to switch from git dependencies to basxconnect, once the somewhat stable packages are released.

Reply:
Right. I think we should try to build a release process with github workflows on the future.